src/components/search/GlobalSearchField.js

```diff
@@ -118,34 +120,35 @@ const SearchOption = React.forwardRef((props, ref) => {
     const classes = useStyles();
     const theme = useTheme();
     return (
-        <ListItem
-            alignItems="flex-start"
-            dense={true}
-            divider={true}
-            href={href}
-            onClick={onClick}
-            ref={ref}
-            id={id}
-        >
-            <ListItemIcon className={classes.icon}>{icon}</ListItemIcon>
-            <ListItemText
-                primary={
-                    <Highlighter search={searchTerm}>{primary}</Highlighter>
-                }
-                primaryTypographyProps={{
-                    variant: "body1",
-                    color: theme.palette.info.primary,
-                }}
-                secondary={
-                    <Highlighter search={searchTerm}>{secondary}</Highlighter>
-                }
-                secondaryTypographyProps={{
-                    variant: "caption",
-                    wrap: "true",
-                    color: theme.palette.info.primary,
-                }}
-            />
-        </ListItem>
+        <>
+            <span className={classes.optionIcon}>{icon}</span>
+            <MuiLink
+                href={href}
+                onClick={onClick}
+                ref={ref}
+                id={id}
+                underline="none"
+                color="inherit"
+            >
+                <div className={classes.optionDiv}>
+                    <Typography
+                        variant={"subtitle2"}
+                        color={theme.palette.info.primary}
+                    >
+                        <Highlighter search={searchTerm}>{primary}</Highlighter>
+                    </Typography>
+                    <Typography
+                        variant={"caption"}
+                        wrap={true}
+                        color={theme.palette.info.primary}
+                    >
+                        <Highlighter search={searchTerm}>
+                            {secondary}
+                        </Highlighter>
+                    </Typography>
+                </div>
+            </MuiLink>
+        </>
```

Comment:
Custom option renderer to eliminate the console warning -DOM validation - <li> cannot be child of <li>